### PR TITLE
Keep default per-operation samplers separate

### PR DESCRIPTION
**Pull request by tjb900**
Test to demonstrate, and one possible fix for, #273

### CODE REVIEW

File: jaeger_client/sampler.py [hunk 1]
```diff
@@ -280,14 +281,19 @@ def __init__(self, strategies, max_operations):
     def is_sampled(self, trace_id, operation=''):
         sampler = self.samplers.get(operation)
         if not sampler:
-            if len(self.samplers) >= self.max_operations:
+            sampler = self.per_operation_default_samplers.get(operation)
+        if not sampler:
+            if (
+                len(self.samplers) + len(self.per_operation_default_samplers) >=
```
Comment (tjb900) [inline jaeger_client/sampler.py]: note for reviewers: with the current configuration flake8 complains whether the line break is either before _or_ after this operator.  I think this means that either W503 or W504 needs to be disabled depending on the project's preference, or `ignore` should be changed to `extend-ignore`?   https://gitlab.com/pycqa/flake8/issues/466

**Comment by tjb900** [comment]
Thanks for the review.

> Should B still apply previously configured probability, or should it switch to the default?

Hmmm yes good point.  The intent of the modified strategies file is presumably that all operations not named explicitly should use the default, so I guess they should switch back to that.  If that's your thinking too, I'll update to make that happen.

File: jaeger_client/sampler.py (hunk 1 above)
Comment (tjb900) [reply jaeger_client/sampler.py]: @yurishkuro Any thoughts on which way to go on this one?

Comment (yurishkuro) [reply jaeger_client/sampler.py]: You can extract the longest expression into a variable.

Comment (tjb900) [reply jaeger_client/sampler.py]: I guess I meant more in terms of fixing the mutually contradictory configuration of the style checking in CI.  As you say though, I can certainly work around it in this case.

**Pull request closed without merging**